### PR TITLE
Add many new modules

### LINKED ISSUE #1: Add more modules to search.nuschtos.de
- [x] https://gitea.c3d2.de/c3d2/nix-config/src/branch/master/hosts/nixos-misc/default.nix#L50-L80
- [x] look at my flake inputs
- [x] look at dd-ix flake inputs

### PR DESCRIPTION
**Pull request by SuperSandro2000**
Closes #1

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -115,13 +335,62 @@
...
+                  name = "sflow_exporter";
+                  urlPrefix = "https://github.com/dd-ix/sflow_exporter/tree/main/";
+                }
+                # sops-nix
+                {
+                  modules = [ sops-nix.nixosModules.default ];
+                  name = "sops-nix";
+                  urlPrefix = "https://github.com/Mic92/sops-nix/blob/master/";
+                }
+                # ticker
```
Comment (SuperSandro2000) [inline flake.nix:382]: and this

File: flake.nix [hunk 2]
```diff
@@ -115,13 +335,62 @@
...
+                          organizationName = "Example Corp";
+                          domain = "example.com";
+                        };
+                      };
+                    }
+                  ];
+                  name = "simple-nixos-mailserver";
+                  urlPrefix = "https://gitlab.com/simple-nixos-mailserver/nixos-mailserver/-/blob/master/";
+                }
+                # sflow-exporter
```
Comment (SuperSandro2000) [inline flake.nix:370]: or this

File: flake.nix [hunk 3]
```diff
@@ -84,10 +260,45 @@
...
+                # ifdyndnsd
+                {
+                  modules = [
+                    ifdyndnsd.nixosModule
+                    { _module.args = { inherit pkgs; }; }
+                  ];
+                  name = "ifdyndnsd";
+                  urlPrefix = "https://github.com/astro/ifdyndnsd/blob/main/";
+                }
+                # ixp-manager.nix
```
Comment (SuperSandro2000) [inline flake.nix:276]: and this

File: flake.nix [hunk 4]
```diff
@@ -84,10 +260,45 @@
                 }
                 # ifstate.nix
                 {
-                  modules = [ ifstate.nixosModules.ifstate ];
+                  modules = [ ifstate.nixosModules.default ];
                   name = "IfState.nix";
                   urlPrefix = "https://codeberg.org/m4rc3l/ifstate.nix/src/branch/main/";
                 }
+                # ifdyndnsd
```
Comment (SuperSandro2000) [inline flake.nix:267]: or this

File: flake.nix [hunk 5]
```diff
@@ -49,24 +152,97 @@
...
+                  name = "buzzrelay";
+                  urlPrefix = "https://github.com/astro/buzzrelay/blob/main/";
+                }
+                # c3d2-user-module
+                {
+                  modules = [ c3d2-user-module.nixosModule ];
+                  name = "c3d2-user-module";
+                  urlPrefix = "https://gitea.c3d2.de/c3d2/nix-user-module/src/branch/master/";
+                }
+                # crowdsec
```
Comment (SuperSandro2000) [inline flake.nix:232]: or those

**Review by SuperSandro2000 — COMMENTED** [review]
we should decide, if the ones I marked with a comment or comment range are significant enough to be kept.

File: flake.nix (hunk 3 above)
Comment (MarcelCoding) [reply flake.nix:263]: I would say not - it is a management software for IXP's and It's not even from us - is just a fork and I am also not maintaining it very well - an update is still pending...

File: flake.nix [hunk 6]
```diff
@@ -49,24 +138,93 @@
...
+                # authentik
+                {
+                  modules = [
+                    authentik.nixosModules.default
+                    { _module.args = { inherit pkgs; }; }
+                  ];
+                  name = "authentik-nix";
+                  urlPrefix = "https://github.com/nix-community/authentik-nix/blob/main/";
+                }
+                # buzzrelay
```
Comment (SuperSandro2000) [inline flake.nix:205]: maybe we should drop buzzrelay, it is really small

**Comment by SuperSandro2000** [comment]
That is not scaling well 
image

**Pull request merged**